### DISCUSSION TITLE
How to convert a mesh of abitrary genus to 0 with CGAL library?

### DISCUSSION THREAD

**Discussion by lxmedai** [CGAL/cgal#7707] [General]
_I wonder if CGAL provides such an algorithm that can fill topological holes of a mesh._ I need to do this as my downstream code can only work with genus-0 mesh. 

## Environment
* Operating system (Windows/Mac/Linux, 32/64 bits): windows-64
* Compiler: visual studio 17 2022
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: 5.6
* Boost version: 1.83.0
* Other libraries versions if used (Eigen, TBB, etc.): -

**Comment by stla** [comment]
If you want to fill the holes, there is `PMP::triangulate_and_refine_hole`.
